features/step_definitions/login_steps.rb

```diff
@@ -1,14 +1,54 @@
 Given(/^the login page$/) do
-  #put your code here
-  pending
+  visit '/'
 end
 
+Name = 'Super Max'
+
 When(/^I log in with proper credentials$/) do
-  #put your code here
-  pending
+  page.find_by_id('username').set Name
```

Comment:
See the Capybara documentation. You could clean this up quite a bit without modifying the underlying html.

https://github.com/jnicklas/capybara